snowplow-tracker/src/main/java/com/snowplowanalytics/snowplow/controller/TrackerController.kt

```diff
@@ -116,4 +120,23 @@ interface TrackerController : TrackerConfigurationInterface {
      * The tracker will start tracking again.
      */
     fun resume()
+
+    /**
+     * Adds user and session information to a URI.
+     *
+     * For example, calling decorateLink on `appSchema://path/to/page` will return:
+     *
+     *      `appSchema://path/to/page?_sp=userId.timestamp.sessionId.appId.platform.domainUserId`
+     *
+     * @param uri The URI to add the query string to
+     * @param parameters The parameters to include in the query string (defaults to all)
```

Comment:
Just a note to also make it clear that this only relates to the optional (extended) parameters, userID and timestamp are set regardless.